sdk/communication/Azure.Communication.CallingServer/src/Models/CallConnectionProperties.cs

```diff
@@ -0,0 +1,46 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+
+namespace Azure.Communication.CallingServer
+{
+    /// <summary> The Details of the call. </summary>
+    public class CallConnectionProperties
+    {
+        /// <summary> Initializes a new instance of CallConnectionProperties. </summary>
+        /// <param name="callConnectionPropertiesInternal">The call connection properties internal.</param>
+        internal CallConnectionProperties(CallConnectionPropertiesInternal callConnectionPropertiesInternal)
+        {
+            /*
+            CallLegId = callConnectionPropertiesInternal.CallLegId;
+            Source = CommunicationIdentifierSerializer.Deserialize(callConnectionPropertiesInternal.Source);
+            Target = CommunicationIdentifierSerializer.Deserialize(callConnectionPropertiesInternal.Target);
+            AlternateCallerId = callConnectionPropertiesInternal.AlternateCallerId == null ? null : new PhoneNumberIdentifier(callConnectionPropertiesInternal.AlternateCallerId.Value);
+            CallConnectionState = callConnectionPropertiesInternal.CallConnectionState;
+            Subject = callConnectionPropertiesInternal.Subject;
+            CallbackUri = new Uri(callConnectionPropertiesInternal.CallbackUri);
+            */
+            var TODO = callConnectionPropertiesInternal;
```

Comment:
[nit] todo?

Reply:
This is reminiscence of previous work I forgot to delete. Removed.